test/unittests/common/commonFormat_test.cpp

```diff
@@ -31,15 +31,15 @@
 
```

Comment:
Ah no, lots of dirs are still missing in unit tests.
See the commented lines in script style_guide_in_makefile.sh